core/creator/src/main/java/io/quarkus/creator/phase/nativeimage/NativeImagePhase.java

```diff
@@ -687,6 +697,9 @@ public boolean set(NativeImagePhase t, PropertyContext ctx) {
                     case "additional-build-args":
                         t.setAdditionalBuildArgs(Arrays.asList(value.split(",")));
```

Comment:
I think you can achieve the same using additional build args.

Why do you prefer separated configuration option `reportExceptionStackTraces` ?